spec/system/pages/home_spec.rb

```diff
@@ -95,125 +87,21 @@
-
-        ra = create(:registration_agency, state: ore)
-        mechanic = create(:occupation_standard, :with_work_processes, :with_data_import, registration_agency: ra, title: "Mechanic")
-        hr = create(:occupation_standard, :with_work_processes, :with_data_import, registration_agency: ra, title: "HR")
-        create(:occupation_standard, :with_work_processes, title: "Pipe Fitter")
-
-        visit home_page_path
-
-        expect(page).to have_link "Oregon", href: occupation_standards_path(state_id: ore.id)
-        within("#oregon") do
-          expect(page).to have_text "2 Apprenticeships"
-        end
-
-        click_on "Oregon"
-
-        expect(page).to have_link "Mechanic", href: occupation_standard_path(mechanic)
-        expect(page).to have_link "HR", href: occupation_standard_path(hr)
-        expect(page).to_not have_link "Pipe Fitter"
-      end
-
-      it "displays Installation etc. industry box" do
-        allow(State).to receive(:find_by).and_return(build_stubbed(:state))
-
-        mechanic = create(:occupation_standard, :with_work_processes, :with_data_import, onet_code: "49-1234", title: "Mechanic")
-        hr = create(:occupation_standard, :with_work_processes, :with_data_import, onet_code: "49-5678", title: "HR")
+      it "displays popular industries box" do
+        hc_industry = create(:industry, name: "Healthcare Support")
+        rn = create(:occupation_standard, :with_work_processes, :with_data_import, onet_code: "31-1234", title: "RN", industry: hc_industry)
+        hr = create(:occupation_standard, :with_work_processes, :with_data_import, onet_code: "31-5678", title: "HR", industry: hc_industry)
         create(:occupation_standard, :with_work_processes, onet_code: "35-1234", title: "Pipe Fitter")
 
         visit home_page_path
 
-        expect(page).to have_link "Installation, Maintenance, and Repair", href: occupation_standards_path(q: "49-")
-        within("#industry-1") do
-          expect(page).to have_text "2 Apprenticeships"
-        end
-
-        click_on "Installation"
-
-        expect(page).to have_link "Mechanic", href: occupation_standard_path(mechanic)
-        expect(page).to have_link "HR", href: occupation_standard_path(hr)
-        expect(page).to_not have_link "Pipe Fitter"
-      end
-
-      it "displays Healthcare Support industry box" do
-        allow(State).to receive(:find_by).and_return(build_stubbed(:state))
-
-        mechanic = create(:occupation_standard, :with_work_processes, :with_data_import, onet_code: "31-1234", title: "Mechanic")
-        hr = create(:occupation_standard, :with_work_processes, :with_data_import, onet_code: "31-5678", title: "HR")
-        create(:occupation_standard, :with_work_processes, onet_code: "35-1234", title: "Pipe Fitter")
-
-        visit home_page_path
-
-        expect(page).to have_link "Healthcare Support", href: occupation_standards_path(q: "31-")
```

Comment:
Can we check for "Healthcare Support" text?

Reply:
For this text check are you wanting me to add that expectation for the new view we've navigated to (not the home page)?

Reply:
Is the "Healthcare Support" text supposed to be on the search results page?